rest/server_context.go

```diff
@@ -652,6 +652,13 @@ func dbcOptionsFromConfig(sc *ServerContext, config *DbConfig, dbName string) (d
 		CompactInterval:           compactIntervalSecs,
 		SgReplicateEnabled:        sgReplicateEnabled,
 	}
+
+	if sc.config.ActiveReplicatorPingInterval != nil {
+		contextOptions.ActiveReplicatorPingInterval = time.Duration(*sc.config.ActiveReplicatorPingInterval) * time.Second
+	} else {
+		contextOptions.ActiveReplicatorPingInterval = DefaultActiveReplicatorPingInterval
+	}
```

Comment:
Just a nitpick, but I've been trying to switch over our config code to this convention - it feels clearer.  Let me know if you disagree.

```suggestion

	contextOptions.ActiveReplicatorPingInterval = DefaultActiveReplicatorPingInterval
	if sc.config.ActiveReplicatorPingInterval != nil {
		contextOptions.ActiveReplicatorPingInterval = time.Duration(*sc.config.ActiveReplicatorPingInterval) * time.Second
	} 
```